container-scan/action.yaml

```diff
@@ -87,9 +142,11 @@ runs:
       uses: thollander/actions-comment-pull-request@v2
```

Comment:
Maybe we can consider skipping comment on `workflow_dispatch` so adding a condition here might help as well ? 
```
if: github.event_name == 'pull_request'